scripts/bench_python_cpp_algos.py

```diff
@@ -292,10 +292,10 @@ def run_all_gap(args):
         },
     ]
 
-    def load_graph(graph_path):
+    def load_graph(graph_path, edge_properties=None):
```

Comment:
Do you want to limit `node_properties` as well? This should be done throughout if you want to.